internal/utils/utils.go

```diff
@@ -191,14 +191,18 @@ const defaultShareDir = "/usr/share/logging"
 func GetShareDir() string {
 	// shareDir is <repo-root>/files - try to find from working dir.
 	const sep = string(os.PathSeparator)
-	const repoRoot = sep + "cluster-logging-operator" + sep
+	const repoRoot = sep + "cluster-logging-operator"
 	wd, err := os.Getwd()
 	if err != nil {
+		log.V(3).Error(err, "Error determining share directory.  Returning default", "default", defaultShareDir)
 		return defaultShareDir
 	}
+	log.V(3).Info("GetShareDir", "workingdir", wd)
```

Comment:
This seems very verbose. Does it make sense to remove it or increase the verbosity needed?